src/services/shims.ts

```diff
@@ -274,10 +274,7 @@ module ts {
         }
 
         public getDefaultLibFileName(options: CompilerOptions): string {
-            // Shim the API changes for 1.5 release. This should be removed once
-            // TypeScript 1.5 has shipped.
-            return "";
-            //return this.shimHost.getDefaultLibFileName(JSON.stringify(options));
+            return this.shimHost.getDefaultLibFileName(JSON.stringify(options));

```

Comment:
Just discussed with Mohamed. I was going to add the typeof check for getDefaultLibFileName to ensure it is backward compatible with the previous lower-cased version (getDefaultLibFilename). Case does not appear to matter for the shimHost though as both work (they return typeof "unknown"). Using a completely different method that isn't on shimHost would return typeof "undefined".